fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -3828,7 +3828,7 @@ protected CompletableFuture<Void> rebuildIndexes(@Nonnull Map<Index, List<Record
                             ),
                             exception -> {
                                 // If there is any issue, simply mark the index as disabled without blocking checkVersion
-                                logExceptionAsWarn(KeyValueLogMessage.build("unable to build index",
+                                logExceptionAsWarn(KeyValueLogMessage.build("unable to rebuild or mark index",
```

Comment:
I thought that we may be able to add some more info to the already existing message (e.g. for the cause of the failure)

Reply:
✅